ckanext/grace_period/plugin.py

```diff
@@ -50,10 +48,10 @@ def _is_resource_available(self, pkg, res):
             except ValueError:
                 is_available = False
             else:
-                is_available = dt > datetime.now()
+                is_available = dt < datetime.now()
 
                 # https://docs.ckan.org/en/2.9/maintaining/authorization.html#dataset-collaborators
-                if is_available and \
+                if not is_available or \
```

Comment:
Not sure about that:
hypotesis: 
1) the grace period is over
2) collabs are enabled
3) user is not a collaborator

--> in line 51 is_available = true
--> the if in 54 is evaluated to true bc the OR with the allow_dataset_collaborators option is true
--> in line 56 the user is denied access bc it is not a collaborator

so the user can't see the resrouce even if it's now public

Reply:
User already denied by date.
This condition tries to grant access in case the user is a collaborator
The code looks a bit messy.  Needs to replace OR with AND to prevent additional checks